README.md

```diff
@@ -197,7 +197,73 @@ data-validation (--verbose or -v) (--log-level or -ll) validate row
   [--filter-status or -fs STATUSES_LIST]
                         Comma separated list of statuses to filter the validation results. Supported statuses are (success, fail). If no list is provided, all statuses are returned.
 ```
+#### Generate Table Partitions for Large Table Row Validations
 
+Below is the command syntax for Generating Tabe Partitions for performing Row Validation on large Tables.
+The following command splits Row Validation into multiple steps by generating multiple Config files which can then be run in Parallel or independently.
```

Comment:
```suggestion
The command generates and stores multiple YAML configs that represent chunks of the large table using filters (`WHERE partition_key > X AND partition_key < Y`). You can then run the configs in the directory serially with the `data-validation configs run --config-dir PATH` command as described [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator#yaml-configuration-files).
```

Reply:
Does this PR support GCS based config-dir paths?

Reply:
Done